### PR TITLE
[wip] hasKeypaths additional documentation example

### PR DESCRIPTION
**Pull request by cflynn07**
hasKeypaths README documentation did not have an example for a popular use case of hasKeypaths, verifying object has [0..n] defined required properties. Existing examples only demonstrated asserting 1 keypath.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -283,10 +283,18 @@ hasKeypaths(obj, { 'foo.bar': barObj }, true);   // true
 hasKeypaths(obj, { 'foo.bar': barObj }, false);  // false
 hasKeypaths(obj, { 'foo.bar': obj.foo }, false); // true
 hasKeypaths(obj, ['foo.bar'], false);            // true, uses [hasOwnProperty vs in](http://stackoverflow.com/questions/13632999/if-key-in-object-or-ifobject-hasownpropertykey)
+
 // use it with find, findIndex, or filter!
 var arr = [obj, { b: 1 }, { c: 1 }];
 find(arr, hasProps({ 'foo.bar.qux':1 })); // { foo: { bar: { qux: 1 } } }
 find(arr, hasProps(['foo.bar.qux']));     // { foo: { bar: { qux: 1 } } }
```
Comment (tjmehta) [inline README.md]: while you're in there, can you change 'hasProps' -> 'hasKeypaths'

File: README.md [hunk 2]
```diff
@@ -283,10 +283,18 @@ hasKeypaths(obj, { 'foo.bar': barObj }, true);   // true
...
+
+// use it to verify options object has required properties
+var opts = {
+  host: 'localhost',
+  port: '3333'
+};
+hasKeypaths(opts, ['host', 'port']); // true
```
Comment (tjmehta) [inline README.md]: can you use a keypath here (with a dot)

**Comment by cflynn07** [comment]
How does that look?

**Comment by tjmehta** [comment]
Thank you!

**Pull request merged**